Nos/Views/Components/NosToggle.swift

```diff
@@ -0,0 +1,23 @@
+import SwiftUI
+
+struct NosToggle: View {
```

Comment:
I'd suggest a doc comment -- maybe something like this.
```suggestion
/// A toggle with the tint color set to green.
/// - Note: Fixes [#1251](https://github.com/planetary-social/nos/issues/1251)
struct NosToggle: View {
```

Reply:
`/// - Note: Fixes [#1251](https://github.com/planetary-social/nos/issues/1251)`

@joshuatbrown , why do we want to include this in the file?

Reply:
We don't do this for all bug tickets, but it's a nice way to explain why some changes were made. Personally I think the link would be more useful right above the line `.tint(.green)`, because I could see myself asking "why are we changing the tint color down here?" But it's fine here too.